lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -1199,3 +1199,16 @@ function dosomething_campaign_node_validate($node, $form, &$form_state) {
     }
   }
 }
+
+  /**
+   * @param object $campaign
+   *   Loaded campaign node to check if given feature is on
+   * @param string $feature
+   *   Checking if this featuer is on
+   */
+  function dosomething_campaign_feature_on($campaign, $feature) {
```

Comment:
yais!

another thing we might want to pull into this is if the feature is enabled for a more than US experiences see #6861